<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/vlad/Downloads/metrou.ico</normaloff>C:/Users/vlad/Downloads/metrou.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resultBar">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-1</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignRight';
border-bottom: 1px solid gray;

	background-color: white;
    color: #333333;
    font-size: 28px;
    padding: 15px;
    border: 2px solid #cccccc;
    border-radius: 5px;
    margin: 10px;
    text-align: right;
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>160</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>160</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>160</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>160</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>220</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>340</y>
      <width>131</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: #333;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #ccc;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #e0e0e0;
}

QPushButton:pressed {
    background-color: #d0d0d0;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}

QPushButton.operation {
    background-color: #ffcc00;
    color: white;
    font-weight: bold;
}

QPushButton.operation:hover {
    background-color: #ffbb00;
}

QPushButton.operation:pressed {
    background-color: #ffaa00;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dotButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>340</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_minusButton">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ACButton">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percentButton">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slashButton">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>100</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #ffcc00;
    color: white;
    font-size: 18px;
    padding: 15px;
    border: 1px solid #cc9a00;
    border-radius: 5px;
    margin: 5px;
    transition: background-color 0.3s, box-shadow 0.3s;
}

QPushButton:hover {
    background-color: #ebba34;
}

QPushButton:pressed {
    background-color: #ebb134;
    box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.2);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
